cli_test/helpers/utils.go

```diff
@@ -0,0 +1,10 @@
+package helpers
```

Comment:
One file for just an exported function seems a bit an overkill. I'd move this into `init.go` honestly.

Reply:
This will have 3-4 functions by the time I finish tests, so I feel it's better to seperate utils